tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -48,17 +54,29 @@ class Meta:
         extra_kwargs = {"password": {"write_only": True}}
 
 
-class SetUserProfileSerializer(serializers.ModelSerializer):
+class UserProfileSerializer(serializers.ModelSerializer):
     """Serializer used for setting a user's profile."""
```

Comment:
Using a more general name for this, because it is now used for both authorization-check and set-profile